src/XFExpandableListView/Controls/ExpandableListView.cs

```diff
@@ -229,16 +235,19 @@ public virtual async Task Collapse(int position)
             /* Hard operations must not affect the UI Thread */
             await Task.Run(() =>
             {
-                foreach (var item in expandableGroup)
+                Device.BeginInvokeOnMainThread(() =>
```

Comment:
Same here, just get it out from the Task.Run

Reply:
Removed